### PR TITLE
FIX: @W-16371174@: Fix bug with config commands case insensitivity

### PR DESCRIPTION
**Pull request by stephen-carter-at-sf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	"bugs": "https://github.com/forcedotcom/sfdx-scanner/issues",
 	"dependencies": {
 		"@oclif/core": "^3.3.2",
-		"@salesforce/code-analyzer-core": "0.20.1",
```
Comment (stephen-carter-at-sf) [inline package.json:9]: This is the main fix

File: src/lib/models/ConfigModel.ts [hunk 2]
```diff
@@ -23,7 +23,6 @@ export interface ConfigModel {
 }
 
 export class AnnotatedConfigModel implements ConfigModel {
-	private readonly config: CodeAnalyzerConfig; // TODO: It would be nice if we updated the CodeAnalyzer (in our core module) to just return its CodeAnalyzerConfig with a getter so we didn't need to pass it around
```
Comment (stephen-carter-at-sf) [inline src/lib/models/ConfigModel.ts:26]: With the other enhancements that I just pulled in, I was able to accomplish this TODO item as well.

File: src/lib/viewers/ResultsViewer.ts [hunk 3]
```diff
@@ -193,11 +193,6 @@ function sortViolations(violations: Violation[]): Violation[] {
 	});
 }
 
-// TODO: We should update core module to have this function directly on the Violation object and then remove this helper
```
Comment (stephen-carter-at-sf) [inline src/lib/viewers/ResultsViewer.ts:196]: With the other enhancements that I just pulled in, I was able to accomplish this TODO item as well.

File: package.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
...
+		"@salesforce/code-analyzer-core": "0.20.2",
```
Comment (stephen-carter-at-sf) [inline package.json:9]: This is the main fix

File: messages/shared.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # label.command-state
 
-Developer Preview
+Beta
```
Comment (stephen-carter-at-sf) [inline messages/shared.md:3]: [FIX-AS-YOU-GO item] We don't want to forget to update this to say "Beta" instead of "Developer Preview" now.

**Review by jfeingold35 — APPROVED** [review]

**Pull request merged**